File: largest-number/largest-number.js
var largestNumber = function (nums) {
    for (let i = 0; i < nums.length; i++) {
        j = i + 1;

        while (j < nums.length && i < nums.length) {
            ij = nums[i].toString() + nums[j].toString()
            ji = nums[j].toString() + nums[i].toString()

            if (parseInt(ij) < parseInt(ji)) {
                temp = nums[i]
                nums[i] = nums[j]
                nums[j] = temp
            }
            j += 1
        }
    }

    res = ""
    nums.map((e) => {
        res += e.toString()
    })

    return parseInt(res) == 0 ? "0" : res
};

File: largest-number/largest-number.py
class Solution(object):
    def largestNumber(self, nums):        
        for i in range(len(nums)):
            j = i + 1
            
            while i < len(nums) and j < len(nums):
                ij = str(nums[i]) + str(nums[j])
                ji = str(nums[j]) + str(nums[i])
                
                if int(ij) < int(ji):
                    nums[j], nums[i] = nums[i], nums[j]
                
                j += 1 
                
        res = "".join(map(str, nums))
        
        return "0" if int(res) == 0 else res